x-pack/plugins/maps/kibana.json

```diff
@@ -25,8 +25,7 @@
     "mapsEms",
     "savedObjects",
     "share",
-    "presentationUtil",
-    "sharedUX"
```

Comment:
This removes sharedUX from requiredPlugins, Can you also remove `sharedUX` from MapsPluginStartDependencies?